core/templates/dev/head/pages/base.html

```diff
@@ -183,6 +182,8 @@ <h2>
     <script src="/templates/dev/head/services/HtmlEscaperService.js"></script>
     <script src="/templates/dev/head/services/TranslationFileHashLoaderService.js"></script>
     <script src="/templates/dev/head/services/RteHelperService.js"></script>
+    <script src="/templates/dev/head/services/StateRulesStatsService.js"></script>
+    <script src="/templates/dev/head/services/ConstructTranslationIdsService.js"></script>
```

Comment:
Why are you adding these two files here?

Reply:
Sorry for that too. I will fix this